### DISCUSSION TITLE
Add UUID v7 to Sequel

### DISCUSSION THREAD

**Discussion by wkdjgf534** [jeremyevans/sequel#2240] [General]
Hi, Recently, I was looking for a way to add `UUID v7` for a RoR project and discovered that SecureRandom has supported it since version 0.3.0 (Ruby 3.3.0). You can check the code [here](https://github.com/ruby/securerandom/blob/master/lib/random/formatter.rb#L179). Is it a good time to add the method' generate_uuuid_v7`, which generates UUID v7 for Sequel?

p.s. I suppose, there a lot of projects still use the Ruby version less than 3.3.0. In this case, I suggest adding checking for `SecureRadom` version equal to or higher than '0.3.0'.

```ruby
require 'securerandom'

def generate_uuid_v7
  gem_version = Gem.loaded_specs['securerandom'].version.to_s

  raise an error if gem_version < '0.3.0' 
  SecureRandom.uuid_v7
end
```

**Comment by jeremyevans** [comment]
Nothing in Sequel would use this method, so I don't think it makes sense to add it.  It may be useful for users of Sequel who work with applications that use UUIDs, but those users should just call `SecureRandom.uuid_v7` themselves, there does not seem to be a point of defining a method to do that (especially if it requires a bunch of checks before doing that).
